README.md

```diff
@@ -198,7 +276,7 @@ Path to file in Fluent Bit config should be well regulated. Fluent Bit Operator
 
 ## Custom Parser
 
-To enable parsers, you must set the value of `FluentBitConfig.Spec.Service.ParsersFile` to `parsers.conf`. Your custom parsers will be included into the built-in parser config via `@INCLUDE /fluent-bit/config/parsers.conf`. Note that the parsers.conf contains a few built-in parsers, for example, docker. Read [parsers.conf](https://github.com/kubesphere/fluent-bit/blob/v1.6.2-reload/conf/parsers.conf) for more information.
+To enable parsers, you must set the value of `FluentBitConfig.Spec.Service.ParsersFile` to `parsers.conf`. Your custom parsers will be included into the built-in parser config via `@INCLUDE /fluent-bit/config/parsers.conf`. Note that the parsers.conf contains a few built-in parsers, for example, docker. Read [parsers.conf](https://github/kubesphere/fluent-bit/blob/v1.6.2-reload/conf/parsers.conf) for more information.
```

Comment:
The url https://github/kubesphere/fluent-bit/blob/v1.6.2-reload/conf/parsers.conf was updated incorrectly and  outdated can could be changed to https://github.com/kubesphere/fluentbit-operator/blob/master/conf/parsers.conf

Reply:
I have made the changes accordingly.